app/views/static_pages/index.html.slim

```diff
@@ -0,0 +1,6 @@
+/ Each view is contained with the outer layout and should begin with a row
+div class="row"
+  div class="col-xs-12 col-md-8 col-md-offset-2"
+    h1 class="text-primary"Future home of the best chess game ever made...
+  div class="col-xs-12 col-md-8 col-md-offset-2"
+    a href="https://github.com/KnightsOfTheRemoteTable/chess_app"Github
```

Comment:
I'd put a space between the close quote (purely for readability) and then give yourself a pat on the back. You know Slim! I'd merge it!

:+1: